packages/cosmo/package.py

```diff
@@ -220,8 +220,14 @@ def install(self, spec, prefix):
     @on_package_attributes(run_tests=True)
     def test(self):
         if '~serialize' in self.spec:
-            subprocess.run(['./test/tools/test_cosmo.py', str(self.spec), prefix], cwd = self.build_directory)
+            try:
+                subprocess.run(['./test/tools/test_cosmo.py', str(self.spec), prefix], cwd = self.build_directory, stderr=subprocess.STDOUT)
+            except subprocess.CalledProcessError:
+                raise ValueError('Testsuite failed')
```

Comment:
```suggestion
            subprocess.run(['./test/tools/test_cosmo.py', str(self.spec), prefix], cwd = self.build_directory, check=True, stderr=subprocess.STDOUT)
```
Probably needs `check=True`.

> If check is true, and the process exits with a non-zero exit code, a CalledProcessError exception will be raised. Attributes of that exception hold the arguments, the exit code, and stdout and stderr if they were captured.

Source: https://docs.python.org/3/library/subprocess.html?highlight=subprocess#subprocess.run